### ISSUE TITLE
Complete documentation

### ISSUE THREAD

**Issue by wbsoft** [wbsoft/python-ly#2]
Now we have basic documentation (auto-extracted from the source) working, it needs to be improved and completed, including roadmap information for the future.

**Comment by uliska** [comment]
Sorry, can't help ATM as I'm busy setting up basic documentation for https:// github.com/openlilylib /scholarly ...

**Comment by wbsoft** [comment]
I'll do this the coming days. I'll restructure the documentation a bit and use smaller html pages. Follow at http://python-ly.readthedocs.org/

**Comment by PeterBjuhr** [comment]
It looks really nice!

**Comment by PeterBjuhr** [comment]
I'll look over the documentation for the `ly.musicxml` subpackage.

One question, is it possible to create links between different sections in the documentation?

**Comment by dliessi** [comment]
> It looks really nice!

Indeed!

**Comment by wbsoft** [comment]
@PeterBjuhr yes, but I have to find out how. Just name the modules when you cross-reference, I'll make references later.
